NuRadioReco/framework/base_station.py

```diff
@@ -76,20 +76,43 @@ def remove_parameter(self, key):
         self._parameters.pop(key, None)
 
     def set_station_time(self, time):
-        if time is None:
-            self._station_time = None
-            return
+        """ 
+        Set the (absolute) time for the station (stored as astropy.time.Time).
+        Not related to the event._event_time.
+        
+        Parameters
+        ----------
+        
+        time: astropy.time.Time or datetime.datetime or float
+            If time is a float it is interpreted as UTC unix timestamp.
+        """
+
         if isinstance(time, datetime.datetime):
-            time_strings = str(time).split(' ')
-            self._station_time = astropy.time.Time('{}T{}'.format(time_strings[0], time_strings[1]), format='isot')
+            self._station_time = astropy.time.Time(time)
+        elif isinstance(time, astropy.time):
+            self._station_time = time
+        elif time == None:
```

Comment:
I guess this is fine? `a == None` (instead of `a is None`) raises an error if `a` is e.g. an array, but I guess there is no risk of that here.